src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static final String Line = "    ____________________________________________________________";
+    private static Task[] tasks = new Task[110];
+    private static int taskSum = 0;
+
+    public static void printList() {
+        Task now;
+        System.out.println(Line);
+        System.out.println("     Here are the tasks in your list:");
+        for(int i=0; i<taskSum; i++){
+            now = tasks[i];
+            System.out.println("     " + (i+1) + "." + now.toString());
+        }
+        System.out.println(Line);
+    }
+    public static void bye() {
+        System.out.println(Line + "\n" + "     Bye. Hope to see you again soon!" + "\n" + Line);
+    }
+    public static void greeting() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println(Line);
+        System.out.println("     Hello! I'm Duke");
+        System.out.println("     What can I do for you?");
+        System.out.println(Line);
+    }
+    public static void main(String[] args) {
```

Comment:
The implementation for this method is quite long. Perhaps it could be refactored further? For example, the handling of each task (todo / deadline / event) could be made different methods. You may also consider making the "happy path" more prominent.